ansible/roles/test/files/ptftests/arista.py

```diff
@@ -558,6 +558,9 @@ def check_last_lacppdu_time(self):
                     .get("Ethernet1", {})\
                         .get("details", {})\
                             .get("lastRxTime")
+        if last_lacp_pdu_time is None:
+            last_lacp_pdu_time = 0
+            self.log("Warn: the lastRxTime does not exist!")
```

Comment:
@nhe-NV, thank you for finding and fixing this. I agree we should handle the case where `lastRxTime` is not returned in the JSON. But setting the PDU time to 0 can lead to
1) Test reporting incorrect values in the report (timestamp can be set to 0, which is actually not true).
2) In the unlikely event when `last_lacppdu_time_before_reboot` gets set to 0 (due to this change), the collection for `last_lacppdu_time_after_reboot` is impacted, as now the code will append any value greater than 0 to the list, and this may give false values for both PDU before and after reboot.

I suggest that instead of storing the value as `0`, do no operation for the case when `lastRxTime` is not returned. In that case 1) we never report timing as 0 and 2) the calculation of lacp-session time will remain based on true values.

Reply:
Hi @vaibhavhd I just changed the code as your suggestion, can you help to check if it is OK to fix it in this way.